### PR TITLE
code rebuild, add evaluate_player for two different checkpoint, add requirements.txt, add parameter list

### PR DESCRIPTION
**Pull request by yangze01**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+theano>=0.7
+lasagne >=0.1
+keras
+numpy
+tensorflow-gpu==2.3.0
```
Comment (DaoZhang0123) [inline requirements.txt:5]: Have you run the code in tf2? I remember the code in tf2 is different with tf1.

**Comment by DaoZhang0123** [comment]
What's the diff in playout400.gif? @yangze01

**Comment by yangze01** [comment]
modify for tensorflow 2.3.0.

**Comment by yangze01** [comment]
> What's the diff in playout400.gif? @yangze01

didn't change the gif. https://raw.githubusercontent.com/junxiaosong/AlphaZero_Gomoku/master/playout400.gif

**Review by DaoZhang0123 — COMMENTED** [review]
> > What's the diff in playout400.gif? @yangze01
> 
> didn't change the gif. https://raw.githubusercontent.com/junxiaosong/AlphaZero_Gomoku/master/playout400.gif

It's strange this file are changed, whatever, let's ignore this issue.

**Comment by DaoZhang0123** [comment]
> modify for tensorflow 2.3.0.

OK.

**Pull request merged**